storage/integration_test.go

```diff
@@ -265,6 +265,9 @@ func TestIntegration_BucketMethods(t *testing.T) {
 	if attrs.LocationType == "" {
 		t.Error("got an empty LocationType")
 	}
+	if attrs.ProjectNumber == 0 {
+		t.Errorf("got a zero ProjectNumber")
+	}
```

Comment:
Can we add this check for creating/deleting with attributes in this same test as well?

Reply:
Done. Added this check in there as well.